README.md

```diff
@@ -1,2 +1,136 @@
 # kubevirt-dpdk-checkup
+
 checkup validating DPDK readiness of cluster, using the Kiagnose engine
```

Comment:
In order to set these the user needs to have the correct security permissions. Can you add something about that?